ngraph/test/op_eval/log_softmax.cpp

```diff
@@ -0,0 +1,48 @@
+//*****************************************************************************
+// Copyright 2020 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//*****************************************************************************
+
+#include <string>
+#include <vector>
+
+#include "gtest/gtest.h"
+
+#include "ngraph/op/log_softmax.hpp"
+#include "ngraph/runtime/host_tensor.hpp"
+#include "ngraph/validation_util.hpp"
+#include "runtime/backend.hpp"
+#include "util/test_tools.hpp"
+
+using namespace std;
+using namespace ngraph;
+
+TEST(op_eval, log_softmax)
```

Comment:
This test is never executed since you haven't added this test file to the appropriate CMakeLists.txt file. When added you get a compilation error originating from line 34 - the LogSoftmax constructor needs the axis attribute to be specified explicitly.

Reply:
BTW, when the test is executed there are result mismatches indicating some problems with the reference implementation:
```
[ RUN      ] op_eval.log_softmax
/home/tdolbniak/projects/openvino/ngraph/test/op_eval/log_softmax.cpp:47: Failure
The difference between result_data[i] and expected_result[i] is 2.1353354454040527, which exceeds 0.000001, where
result_data[i] evaluates to -0.27227067947387695,
expected_result[i] evaluates to -2.4076061248779297, and
0.000001 evaluates to 9.9999999999999995e-07.
/home/tdolbniak/projects/openvino/ngraph/test/op_eval/log_softmax.cpp:47: Failure
The difference between result_data[i] and expected_result[i] is 1.3678795099258423, which exceeds 0.000001, where
result_data[i] evaluates to -0.039726495742797852,
expected_result[i] evaluates to -1.4076060056686401, and
0.000001 evaluates to 9.9999999999999995e-07.
/home/tdolbniak/projects/openvino/ngraph/test/op_eval/log_softmax.cpp:47: Failure
The difference between result_data[i] and expected_result[i] is 1.0000000596046448, which exceeds 0.000001, where
result_data[i] evaluates to 0.59239405393600464,
expected_result[i] evaluates to -0.40760600566864014, and
0.000001 evaluates to 9.9999999999999995e-07.
```

Reply:
Thanks, fixed